cicd/gitops/gitops-release-notes.adoc

```diff
@@ -17,11 +17,13 @@ toc::[]
 
 For an overview of {gitops-title}, see xref:../../cicd/gitops/understanding-openshift-gitops.adoc#understanding-openshift-gitops[Understanding OpenShift GitOps].
 
-include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
-
 include::modules/go-compatibility-and-support-matrix.adoc[leveloffset=+1]
 
+include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
+
 // Modules included, most to least recent
+include::modules/go-release-notes-1-3-1.adoc[leveloffset=+1]
```

Comment:
```suggestion
include::modules/gitops-release-notes-1-3-1.adoc[leveloffset=+1]
```
Any reason for the change from 'gitops' to 'go' in the filename? I suggest keeping the name in sync with the old ones.